source/whats-new.txt

```diff
@@ -0,0 +1,318 @@
+
+Improvements
+```````````
+
+- Documents expected behavior of command errors in a transaction.
+
+.. _c-version-1.27.5:
+
+What's New in 1.27.5
+--------------------
+
+The 1.27.5 release of ``libbson`` and ``libmongoc``, the libraries constituting 
+the C driver, includes several changes.
+
+libbson
+~~~~~~~
+
+The 1.27.5 release of ``libbson`` includes the following changes:
+
+- Fixes large string handling in ``bson_append_utf8``.
+
+libmongoc
+~~~~~~~~~
+
+The 1.27.5 release of ``libmongoc`` includes the following changes:
+
+- Fixes possible build error in environments where ``bool`` or ``_Bool`` is a macro.
+
+.. _c-version-1.27.4:
+
+What's New in 1.27.4
+--------------------
+
+The 1.27.4 release of ``libbson`` and ``libmongoc``, the libraries constituting 
+the C driver, includes several changes.
+
+libbson
+~~~~~~~
+
+The 1.27.4 release of ``libbson`` includes the following changes:
+
+- Fixes build issues with Visual Studio 2013.
+
+libmongoc
+~~~~~~~~~
+
+The 1.27.4 release of ``libmongoc`` includes the following changes:
+
+- Fixes possible crash reading malformed wired protocol message.
+- Fixes build issues with Visual Studio 2013.
+
+.. _c-version-1.27.3:
+
+What's New in 1.27.3
+--------------------
+
+The 1.27.3 release of ``libbson`` and ``libmongoc``, the libraries constituting 
+the C driver, includes several changes.
+
+libbson
+~~~~~~~
+
+The 1.27.3 release of ``libbson`` includes the following changes:
+
+Changes
+```````
+
+- This patch introduces changes surrounding the development processes, 
+  including new code signing and dependency tracking. There are no behavioral or API changes.
+
+libmongoc
+~~~~~~~~~
+
+The 1.27.3 release of ``libmongoc`` includes the following changes:
+
+Fixes
+`````
+
+- Upgrades bundled Zlib from 1.2.3 to 1.3.1.
+
+.. _c-version-1.27.2:
+
+What's New in 1.27.2
+--------------------
+
+The 1.27.2 release of ``libbson`` and ``libmongoc``, the libraries constituting 
+the C driver, includes several changes.
+
+libbson
+~~~~~~~
+
+The 1.27.2 release of ``libbson`` includes the following changes:
+
+Fixes
+`````
+
+- Fixes parsing of colon in timezone.
+- Fixes error on failure to initialize a subdocument in ``bson_iter_visit_all``.
+
+libmongoc
+~~~~~~~~~
+
+The 1.27.2 release of ``libmongoc`` includes the following changes:
+
+Fixes
+`````
+
+- Closes lefotver connections to remote servers.
+
+New Features
+````````````
+
+- Checks environment variable ``MONGO_EXPERIMENTAL_SRV_PREFER_TCP`` to prefer TCP over SRV lookup.
+
+.. _c-version-1.27.1:
+
+What's New in 1.27.1
+--------------------
+
+The 1.27.1 release of ``libbson`` and ``libmongoc``, the libraries constituting 
+the C driver, includes several changes.
+
+libbson
+~~~~~~~
+
+The 1.27.1 release of ``libbson`` includes the following changes:
+
+- Fixes large string handling in ``bson_string_new`` and ``bson_string_append``.
+
+libmongoc
+~~~~~~~~~
+
+No changes since 1.27.0. Version incremented to match the ``libbson`` version.
```

Comment:
S: make this a complete sentence
```suggestion
The version is incremented to match the ``libbson`` version, but there are no changes since 1.27.0.
```

Reply:
Applies to all